packages/cli/src/rest/api.ts

```diff
@@ -46,26 +46,42 @@ export async function validateAuthentication (): Promise<void> {
   }
 }
 
+export function requestInterceptor (config: InternalAxiosRequestConfig) {
+  const { Authorization, accountId } = getDefaults()
+  if (Authorization && config.headers) {
+    config.headers.Authorization = Authorization
+  }
+
+  if (accountId && config.headers) {
+    config.headers['x-checkly-account'] = accountId
+  }
+
+  return config
+}
+
+export function responseErrorInterceptor (error: any) {
+  if (error.response?.status === 408) {
+    throw new Error('Encountered an error connecting to Checkly. Your Internet connection is too slow.')
```

Comment:
We should improve this message. It is not an error while connecting to Checkly but during transferring the data. We should also paraphrase the reason. "This can be triggered by a slow internet connection or a network with high packet loss"

Reply:
@umutuzgur , I changed the error reaso. I also kept "Encountered an error connecting to Checkly.". Is it OK for you, or you want me to remove it and just show the message you sent above.